nf_core/pipeline-template/{{cookiecutter.pipeline_slug}}/docs/configuration/adding_your_own.md

```diff
@@ -58,18 +58,22 @@ docker {
 }
 ```
 
+A test profile comes with the pipeline and is used by the Travis continuous integration (CI) service to test the pipeline for potential errors. Typically, this downloads a small test dataset from [test data repository](https://github.com/nf-core/test-datasets/) and runs the pipeline automatically using docker using [Travis](https://travis-ci.org/nf-core/{{ cookiecutter.pipeline_name }}/) whenever changes are made to the pipeline. Further information on how to add your own test data for a new pipeline can be found at the link mentioned above. 
 
 ### Singularity image
 Many HPC environments are not able to run Docker due to security issues. [Singularity](http://singularity.lbl.gov/) is a tool designed to run on such HPC systems which is very similar to Docker. Even better, it can use create images directly from dockerhub.
-
 To use the singularity image for a single run, use `-profile standard,singularity`. This will download the singularity container from singularity hub dynamically.
 
+
 To specify singularity usage in your pipeline config file, add the following:
 
 ```nextflow
 singularity {
   enabled = true
 }
+process {
+  container = "shub://$wf_container"
```

Comment:
This is out of date now (we don't define `$wf_container` these days). Should be:

```nextflow
singularity.enabled = true
process.container = {"shub://${params.container.replace('nfcore', 'nf-core')}"}
```

Reply:
Or maybe better still:

```
singularity.enabled = true
process.container = params.container
```

Reply:
This is still not in my mind :confused: